pkg/sbom/cyclonedx/unmarshal.go

```diff
@@ -104,7 +104,7 @@ func (c *CycloneDX) UnmarshalJSON(b []byte) error {
 	// Keep the original SBOM
 	c.CycloneDX = &ftypes.CycloneDX{
 		BOMFormat:    bom.BOMFormat,
-		SpecVersion:  bom.SpecVersion,
+		SpecVersion:  strconv.Itoa(int(bom.SpecVersion)),
```

Comment:
I wonder if we should convert `SpecVersion`.
https://github.com/CycloneDX/cyclonedx-go/blob/39a1d8f8f69040a53114e1ea481e48f6d792c05e/cyclonedx.go#L470-L478